GetDKAN/data-catalog-react
# Add Cypress and Testing-Library

This PR aims to move all the Cypress tests from frontend to this repo, but rewrite where possible using testing library to remove the need for classes in the tests. This should allow for tests from components to be partially reused when added to the demo site. 

For a comparison, here are the tests in the home.spec on the frontend repo: https://github.com/GetDKAN/data-catalog-frontend/blob/master/cypress/integration/home.spec.js

Need to finish the CI as it doesn't seem to like the steps I added manually and we can't use dktl as it currently still looks for data-catalog-frontend.

QA Steps:
1. Pull repo locally
1. Run cypress and all tests should be green if you have it connected to a working dkan demo site.
